chef/cookbooks/network/attributes/default.rb

```diff
@@ -27,18 +34,26 @@
     default[:network][:ovs_service] = "openvswitch"
   end
 when "rhel"
-  default[:network][:base_pkgs] = ["bridge-utils",
-                                   "vconfig"]
-  default[:network][:ovs_pkgs] = ["openvswitch",
-                                  "openstack-neutron-openvswitch"]
+  default[:network][:base_pkgs] = [
+    "bridge-utils",
+    "vconfig"
+  ]
+  default[:network][:ovs_pkgs] = [
+    "openvswitch",
+    "openstack-neutron-openvswitch"
+  ]
   default[:network][:ovs_service] = "openvswitch"
 
 else
-  default[:network][:base_pkgs] = ["bridge-utils",
-                                   "vlan"]
-  default[:network][:ovs_pkgs] = ["linux-headers-#{`uname -r`.strip}",
-                                  "openvswitch-datapath-dkms",
-                                  "openvswitch-switch"]
+  default[:network][:base_pkgs] = [
+    "bridge-utils",
+    "vlan"
+  ]
+  default[:network][:ovs_pkgs] = [
+    "linux-headers-#{`uname -r`.strip}",
```

Comment:
To be really correct I would replace this with `"linux-headers-#{node["kernel"]["release"]}",`